laravel-app/resources/views/profile/show.blade.php

```diff
@@ -10,10 +10,29 @@
 				<img class="img-fluid w-50" src="{{ asset('assets/images/microblog-logo.png') }}" alt="Microblog Logo">
 			</div>
 			<div class="col-6 pt-1">
-				<div class="row">
+				<div class="row justify-content-between">
 					<div class="col">
 						<label>Name</label>
-						<p class="fw-bold">{{ $user->user_information->first_name . ' ' . $user->user_information->middle_name . ' ' . $user->user_information->last_name}} </p>
+						<p class="fw-bold">{{
+							$user->user_information->first_name . ' ' . 
```

Comment:
you can create function in the model itself

Reply:
+1

Reply:
done sirs, used accessor in the User model to get the full name attribute of the user instance. Thank you ❤️